src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Link, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { fetchRegistration } from '../../../api/registrationApi';
+import { Registration, RegistrationStatus } from '../../../types/registration.types';
+
+interface DeletedRegistrationInformationProps {
+  registration: Registration;
+}
+
+export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+  const { t } = useTranslation();
+  const duplicateRegistrationQuery = useQuery({
+    enabled: !!registration.duplicateOf,
+    queryKey: ['registration', registration.duplicateOf],
+    queryFn: () => fetchRegistration(registration.duplicateOf ?? ''),
+    meta: { errorMessage: t('feedback.error.get_registration') },
+  });
+
+  const duplicateRegistrationTitle =
+    duplicateRegistrationQuery.data?.entityDescription?.mainTitle ?? registration.duplicateOf;
+  const unpublishingNote = registration.publicationNotes?.find((note) => note.type === 'UnpublishingNote');
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '0.1rem',
+        marginTop: '1rem',
+      }}>
+      <Box
+        sx={{
+          bgcolor: 'publishingRequest.main',
+          display: 'grid',
+          gridTemplateColumns: '1fr auto',
```

Comment:
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/a6df5d86-c4c2-414d-9f72-0ec2659dae5c)
vs
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/991e2edb-04c2-498b-8835-ef7547001139)

```suggestion
          gridTemplateColumns: '1fr auto',
          alignItems: 'center'
```

Dette blir også veldig rart/stygt når det er en lang tekst der da, siden venstrteksten da blir svenvende midt i blokka litt i ingenmannsland. Men fint å ta mer justeringer i det i de kommende PRs som du nevner 👍

Reply:
Ikke tenkt å være slikt for alltid heldigvis da. 
Forhåpentligvis  om ikke så veldig lenge så får vi inn "hvem" og "når" også og da skal det bli litt penere.